### PR TITLE
Do not install mgrctl and mgradm in the server container image

### PR DESCRIPTION
**Pull request by deneb-alpha**
## What does this PR change?

When we prepare our host system for making it ready to manage the Uyuni and  SUSE Multi-Linux Manager containers, we automatically install some additional tools called `uyuni-tools`. In particular, on the server we install `mgrctl` and `mgradm`.

Of course, when we build our container images, we don't want to also install those tools inside the containers.

Currently, when building the container images we also automatically install the release packages, but those packages are also additionally installing back the `uyuni-tools`.

The solution make sure that the final image doesn't include  `mgrctl` and `mgradm` in the container.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite
- Tested manually: See the content of `.packages` in https://build.opensuse.org/projects/systemsmanagement:Uyuni:Master/packages/server-image/repositories/containers/binaries vs https://build.opensuse.org/projects/home:deneb_alpha:branches:systemsmanagement:Uyuni:Master/packages/server-image/repositories/containers/binaries

- [x] **DONE**

## Links

Issue(s): reported by QE squad.

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: containers/server-image/server-image.changes.deneb-alpha.NoToolsOnContainers [hunk 1]
```diff
@@ -0,0 +1 @@
+- Do not install mgrctl and mgradm in the container
```
Comment (srbarrios) [inline containers/server-image/server-image.changes.deneb-alpha.NoToolsOnContainers:1]: Installing and removing is not exactly the same than not installing.
Maybe we can say here, remove unnecessary installed mgrctl and mgradm files.

Also, do we have the same issue with mgrpxy on the proxy container, right?

**Review by srbarrios — APPROVED** [review]
Approving, just a nitpick on the changelog

File: containers/server-image/server-image.changes.deneb-alpha.NoToolsOnContainers (hunk 1 above)
Comment (jordimassaguerpla) [reply containers/server-image/server-image.changes.deneb-alpha.NoToolsOnContainers:1]: looks a bit hacky to me.

**Comment by deneb-alpha** [comment]
It seems that the preferred way to go needs to be https://github.com/uyuni-project/uyuni/pull/9579.
Closing this PR in favor of the other approach.

**Pull request closed without merging**